src/logger.js

```diff
@@ -0,0 +1,20 @@
+const chalk = require('chalk');
+
+const LOG_LEVEL_ENUM = Object.freeze({info: 1, warn: 2, crit: 3});
+const LOG_LEVEL_CHALK_MAP = Object.freeze({info: "green", warn: "yellow", crit: "red"});
+
+const log = module.exports.log = Object.defineProperties({}, {
```

Comment:
Is there some other third-party functionality we can harness for logging? I don't want this PR to explode too much though, feel free to open a task for later.
For example, a critical feature IMO is timestamps being logged, as well as prepending "INFO:" (etc., the log level) for the log lines.

Does `chalk` ensure that if redirected to a file, the colour escape sequences are removed?